auth/contract/store.go

```diff
@@ -79,7 +84,7 @@ func (m TemplateStore) Get(cType Type, language Language, version Version) *Temp
 	return nil
 }
 
-func (m TemplateStore) FindFromRawContractText(rawContractText string) (*Template, error) {
+func (m TemplateMap) FindFromRawContractText(rawContractText string) (*Template, error) {
```

Comment:
It is a bit strange indeed. 
We probably need to refactor this. Question is: should we include this in this PR? Since this "interface" was already in use.